jobs/claim_unused_prls_test.go

```diff
@@ -0,0 +1,295 @@
+package jobs_test
+
+import (
+	"github.com/oysterprotocol/brokernode/jobs"
+	"github.com/oysterprotocol/brokernode/models"
+	"github.com/oysterprotocol/brokernode/services"
+	"time"
+)
```

Comment:
Structured the tests in these two test files a little differently since I'm re-using the same test data throughout the file.